### ISSUE TITLE
Add .plot() to Spectrum

### ISSUE THREAD

**Issue by jspaaks** [matchms/matchms-backup#52]
Add method ``.plot()`` to the ``Spectrum`` class. It should create M/z v. intensity plots like https://github.com/NLeSC/collab-demos/blob/master/demos/emetabolomics/mass-spectrometry-and-liquid-chromatography.md.

Maybe we can use 
- https://github.com/bittremieux/spectrum_utils
- https://www.biorxiv.org/content/10.1101/725036v1.full.pdf

I vote to postpone or forego testing on this one

**Comment by jspaaks** [comment]
See also:
- step 1 in this notebook https://github.com/matchms/notebooks/blob/9a8f94db0250c2cc540b83de87147c08fac34187/tutorial.ipynb and
- my comment https://github.com/matchms/notebooks/pull/12#issue-395224959.

**Comment by jspaaks** [comment]
I tried using specrtum_utils, but its plot function requires that we install, e.g. ``rdkit``. To me, this seemed unnecessary for a simple stem plot. Therefore, I copy-pasted some code from @florian-huber that already made a stem plot using ``matplotlib``.

This is part of https://github.com/matchms/matchms/pull/67. Once that is merged, this issue should be closed.
